internal/impl/azure/shared/service.go

```diff
@@ -121,3 +123,48 @@ func parseConnectionString(input string) (storageAccount, storageAccessKey strin
 	}
 	return accountName, accountKey, nil
 }
+
+// GetStorageClient creates the storage client for the provided credentials
+// TODO: Use a proper config struct for the parameters to avoid mixing them up
+func GetStorageClient(storageConnectionString, storageAccount, storageAccessKey, storageSASToken string) (*azblob.Client, error) {
+	var client *azblob.Client
+	var err error
+	if len(storageConnectionString) > 0 {
+		storageConnectionString := storageConnectionString
+		if strings.Contains(storageConnectionString, "UseDevelopmentStorage=true;") {
+			storageConnectionString = GetEmulatorConnectionString("10000", "10001", "10002")
+		}
+		// The UI doesn't add the AccountName parameter to the connection string for some reason...
+		if !strings.Contains(storageConnectionString, "AccountName=") {
+			storageConnectionString = storageConnectionString + ";" + "AccountName=" + storageAccount
+		}
```

Comment:
Not a big deal but are you sure about this? never seen it happening

Reply:
Dunno why it’s happening in my corporate account. Seems like a really weird omission on their end. I thought it’s quite harmless to add it automatically as a convenience, even though the error it gives should help the user figure out what’s missing.